man/tpm2_activatecredential.1.md

```diff
@@ -14,25 +14,25 @@ key.
 # DESCRIPTION
 
 Verify that the given content is protected with given keyHandle for given
-handle, and then decrypt and return the secret, if any passwd option is
+handle, and then decrypt and return the secret, if any password option is
 missing, assume NULL. Currently only support using TCG profile compliant EK as
 the keyHandle.
 
 # OPTIONS
 
 These options control the object verification:
 
-  * **-H**, **--handle**=_HANDLE_:
-    _HANDLE_ of the object associated with the created certificate by CA.
+  * **-c**, **--context**=_CONTEXT\_OBJECT_:
```

Comment:
I think we need a better long option name... Maybe something like: **--object**=_OBJ\_CTX\_OR\_HANDLE_